.gitlab-ci.yml

```diff
@@ -25,7 +25,7 @@ build:
   stage: build
   image: docker:git
   services:
-    - docker:19.03.5-dind
+    - docker:20.10.8-dind
```

Comment:
Mender-qa is using simply `docker:dind`. Not sure if that's better or worse.

Reply:
Let's use `docker:dind`